src/ldc/eh/msvc.d

```diff
@@ -377,14 +383,14 @@ void msvc_eh_terminate() nothrow
             mov RBX, 0xFFFFFF;
             and RDX, RBX;
             cmp RDX, 0xC48348;          // add ESP,nn  (debug UCRT libs)
-            je L_addESP_found;
+            je L_addESP_found;

```

Comment:
Thanks for the PR, I'll test it over the weekend. All changes starting with this one seem to be different line endings only. Could you please fix that (for the whole file)?